indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/SeekableStreamIndexTask.java

```diff
@@ -179,6 +179,12 @@ public <T> QueryRunner<T> getQueryRunner(Query<T> query)
     return (queryPlus, responseContext) -> queryPlus.run(getRunner().getAppenderator(), responseContext);
   }
 
+  @Nullable
+  public String getStatus()
```

Comment:
Nit: Please add a javadoc to clarify that this is the status is not the same as the final `TaskStatus`.
You may even rename this method to `getCurrentRunnerStatus()` to avoid ambiguity.

Reply:
Added, and changed name accordingly